tests/declare_class/lib/test.js

```diff
@@ -2,7 +2,7 @@ declare class _C {
   foo(): number;
 }
 declare var _module: {
-  C: Class<_C>;
+  C: typeof _C;
```

Comment:
This may generate some traffic in the issue tracker.  *Type* `_C` is not `this`-abstract, so extending `_module.C` under the `Class<_C>` case is no good.  The `typeof` operator exposes *value* `_C` for the work around.  I understand that somebody is currently working toward a solution here....